pyomo/repn/linear.py

```diff
@@ -61,6 +61,10 @@
 _GENERAL = ExprType.GENERAL
 
 
+def _inv2str(val):
+    return f"{val._str() if hasattr(val, '_str') else val}"
+
+
```

Comment:
Because this is exactly the same as lines 173-176 in `pyomo/repn/ampl`, wouldn't it make sense to instead put this in `pyomo/repn/utils`?

Reply:
Possibly, but I didn't really want to advertise / test this as public functionality, but accessing "private" (underscored) objects from outside their module is also frowned upon.  As the function was pretty trivial, I went with duplication.